### ISSUE TITLE
Not working as a deployment

### ISSUE THREAD

**Issue by deepakdhamuria** [maanavshah/stock-market-india#12]
Hello @maanavshah first of all, this is an awesome package. Great job on this.

I tried installing this on AWS and Heroku, but it times out on both these cases. Any idea why and how can we proceed in resolving this? I would like to collaborate and take this further.

**Comment by maanavshah** [comment]
Hi Deepak,

Thanks. the issue is that the URL for NSE has changed. We need to fix it to make it work. I need some time before I can start working on this.

**Comment by deepakdhamuria** [comment]
@maanavshah and @kkirankumar1511 the thing is, this package scrapes data from NSE website as per my understanding. How it does it is by changing the "Referrer" header to NSE website so that the NSE endpoint thinks it to be its own website and sends back data. Now, this would work from Postman or local without any issues, but won't work from a server as the security would strip that header and send it. 

So, if we can figure out how to change that header in a server environment, we would be able to make this working.

In short, this cannot be used for a production implementation.

**Comment by deepakdhamuria** [comment]
I am closing this because I don't think this might be possible to implement this in a server.
